tests/data/duplicate_identities/params/predict_server/predict_server.yml

```diff
@@ -1,7 +1,7 @@
 arch: tfgnn
 log_level: info
 tf_log_level: critical
-tactic_expand_bound: 2  # there are only 3 tactics in this dataset
+tactic_expand_bound: 5  # there are only 3 tactics in this dataset
```

Comment:
I changed this to 5 to test that we can now make the tactic_expand_bound bigger than that number of available tactics.   This sort of thing used to crash.  It doesn't change the results.